edx_proctoring/api.py

```diff
@@ -1804,7 +1804,7 @@ def _get_practice_exam_view(exam, context, exam_id, user_id, course_id):
         student_view_template = 'proctored_exam/ready_to_submit.html'
 
     if student_view_template:
-        context['backend_js'] = provider.get_javascript()
+        context['backend_js_bundle'] = provider.get_javascript()
```

Comment:
nit: I find this name a bit confusing as its just the bundle url, not the javascript itself.